lib/github.js

```diff
@@ -71,6 +71,12 @@ Organization.prototype.getRepositories = function(callback) {
             cli.debug('next page ' + this.nextPageUrl);
             cli.debug('last page ' + this.getLastPage());
 
+            if ( body.message ) {
```

Comment:
I'm not sure, that's why I asked :smiley: . I thought it wasn't be needed because we are checking for the status code up [here](https://github.com/tegon/clone-org-repos/pull/3/files#diff-8f83cf4e554cfef336da48f2169e3704R66). As far as I saw, the lib send the correct status code, but don't set an error on callback when the status code is 401.